### ISSUE TITLE
OE SDK should consider releasing a debug versions of its libraries

### ISSUE THREAD

**Issue by CodeMonkeyLeet** [openenclave/openenclave#2333]
OE SDK currently only releases a single Release with debug info version of the package for developers who do not wish to build the libraries from source. This creates some trade offs in usability as developers may want to take advantage of debug features compiled into OE SDK or its compiled versiosn of 3rdparty libraries like mbedtls (as in the case of #2332).

To make this easier for app devs to consume, we should consider also releasing debug versions of the libraries:

* For Windows .nuget package, the debug versions can usually be included in the same package under a separate folder.
* For Ubuntu .deb packages, the common practice is to provide a separate `*-dbg`/`*-debug` package.

**Comment by AevaOnline** [comment]
Agreed. Debug libraries are standard practice within many open source projects, and I can see the value in producing `*-debug` when a release is built.

**Comment by jhand2** [comment]
For deb packages on Ubuntu -dev packages usually include headers and don't include debug symbols. The normal package is no headers, no symbols, just libs. -debug would be just the stripped symbols in a separate package.
 
As for nuget, generally what I see is a single package with the following layout:

```
root/
   include/
   bin/
      x64/
         debug/
         release/
```
